### ISSUE TITLE
Calling the WalletService.DeleteItem() method results with: RpcException: Unimplemented

### ISSUE THREAD

**Issue by tgdula** [Dentity-Development/dentity-ecosystems-sdk#440]
Calling the `WalletService.DeleteItem()` (or `DeleteItemAsync()`) method results with `RpcException: Unimplemented`. 

**Sample code**:
```
AccountService accountService = new AccountService();
var account = new AccountDetails() { Name = "test" };
var signedIn = await accountService.SignInAsync(account);
var walletService = new WalletService(signedIn);
var item = new JObject { { "foo", "bar" } };
var itemId = await walletService.InsertItemAsync(item);
var deleteRequest = new DeleteItemRequest();
var delete = await walletService.DeleteItemAsync(deleteRequest);
```

**Current result**:
```
Grpc.Core.RpcException: Status(StatusCode=\"Unimplemented\", Detail=\"\")
   at Trinsic.WalletService.DeleteItemAsync(DeleteItemRequest request)
```

**Expected result**: it should be possible to successfully delete an item with `DeleteItem()` / `DeleteItemAsync()`  method

**Environment details**: MS Windows 10 Pro, .NET Core 6.0

**Software version**: Trinsic 1.3.0, Okapi.Net 1.2.3

**Reproduction rate**: 100%

**Comment by tmarkovski** [comment]
Thanks for reporting this @tgdula, this somehow slipped thru the implementation. I'll update this issue once the fix is out.

**Comment by tgdula** [comment]
Thank you @tmarkovski for checking and confirming this issue!

I see my issue is closed, but  there's still only old versions of the library (up to 1.3.0) [in NuGet](https://www.nuget.org/packages/Trinsic). I can't access the one with your fix.

When do you plan to make the new version available, so that I could use it?

**Comment by tmarkovski** [comment]
Sorry, github closed it when the PR got merged. It's not yet deployed. I'll post update when that's out.

**Comment by fundthmcalculus** [comment]
@tmarkovski this fix should be out with the `1.4` version release, no?

**Comment by tmarkovski** [comment]
Correct, this is out with latest sdk
